eventtracking/backends/logger.py

```diff
@@ -45,7 +45,7 @@ def send(self, event):
 class DateTimeJSONEncoder(json.JSONEncoder):
     """JSON encoder aware of datetime.datetime and datetime.date objects"""
 
-    def default(self, obj):  # pylint: disable=method-hidden
+    def default(self, obj):  # lint-amnesty, pylint: disable=arguments-differ, method-hidden
```

Comment:
That is how they got there.     I found that the repo had some lint errors that were causing travis builds to fail, so I switched to using edx-lint and ran the lint-amnesty.   Is there something in the comment above that suggests I did something wrong?   Should I remove the lint-amnesty part of the comment?